src/java/br/grupointegrado/facebug/exception/ValidacaoException.java

```diff
@@ -0,0 +1,13 @@
+package br.grupointegrado.facebug.exception;
+
+/**
+ * Exceção utilizada na validação de dados
+ * 
+ * @author Edvaldo
+ */
+public class ValidacaoException extends RuntimeException {
```

Comment:
Você pode fazer o `extends` para `Exception` direto, assim você obriga quem for usar a tratar o erro.